spec/requests/discourse_code_review/code_review_controller_spec.rb

```diff
@@ -144,7 +146,8 @@
     SiteSetting.code_review_auto_unassign_on_approve = true
     SiteSetting.code_review_allow_self_approval = true
 
-    user = Fabricate(:admin)
+    default_allowed_group = Group.find_by(name: 'staff')
+    user = Fabricate(:admin, groups: [default_allowed_group])
```

Comment:
Does that admin fabricator not automatically add the user to the staff group?

Reply:
No, that's the reason why it was failing. The same thing happens in `discourse-assign`